compiling_everything_linux.md

```diff
@@ -0,0 +1,65 @@
+# Compiling Teeworlds on Linux
+
+**IMPORTANT NOTE:** SDL and Freetype libs are not shipped with Teeworlds 0.7.x. You must download them separately.
+
+**Q: What is BAM?**
+
+Bam is the [build system made by matricks](http://matricks.github.io/bam/) used in Teeworlds.
+
+#### Setup
+1. Use your package manager (apt-get, emerge or whatever is used on your distribution) to install the following (you will need the header files):
+    - gcc
+    - g++
+    - python
+    - alsa (asound)
+    - gl
+    - glu
+    - x11
+    - libsdl
+    - freetype
+2. Download and unzip [Teeworlds-source](https://github.com/teeworlds/teeworlds/releases) or [Teeworlds-latest-source](https://github.com/teeworlds/teeworlds)
+3. Download and unzip [bam](https://github.com/matricks/bam/archive/v0.5.1.tar.gz) to `\teeworldsSource\bam`
```

Comment:
The explanations for the 0.6 and earlier versions put `bam` next to `teeworlds` (probably saner). Better keep it uniform. Commands lower use `../bam/`